client/components/language-chooser/index.jsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import LanguageChooserModal from './language-chooser-modal';
+
+class LanguageChooser extends PureComponent {
+
+	static propTypes = {
+		languages: PropTypes.array.isRequired,
+		valueLink: PropTypes.shape( {
+			value: PropTypes.any,
+			requestChange: PropTypes.func.isRequired
+		} ),
+	}
+
+	constructor( props ) {
```

Comment:
Looks like you could use ESNext property initialisers instead of this constructor.